docs/source/channels.rst

```diff
@@ -0,0 +1,99 @@
+Channels: Futures Shared Between Clients
+===================================
+
+Often it is desirable to respond to events from outside the distributed cluster 
+or to instantiate a new client in order to check on the progress of a set of tasks. 
+The channels feature makes these and many other workflows possible. 
+It also allows the Dask Scheduler to be extended in a clean way using the normal
+Distributed task submission.
+
+Channel:
+--------
+A changing stream of futures shared between clients
+
+Several clients connected to the same scheduler can communicate a sequence
+of futures between each other through shared channels. All clients can
+append to the channel at any time. All clients will be updated when a
+channel updates. The central scheduler maintains consistency and ordering
+of events.
+
+Examples
+--------
+
+Create channels from your Client:
+
+.. code-block:: python
+>>> client = Client('scheduler-address:8786')
+>>> chan = client.channel('my-channel')
```

Comment:
I expect sphinx to be upset with this formatting.  I usually have to do have an extra line between `code-block` and the code.  I also have to indent by three spaces.  I'm often wrong about sphinx though and would be happy is this was valid.

In case I'm not wrong, you should be able to build docs locally by 

```
cd docs
make html
```